example/src/index.ts

```diff
@@ -1,2 +1,4 @@
 const foo: any = "";
 console.log(foo);
+const CamelCased = "abc";
```

Comment:
Technically camel cased is `camelCased` and not `CamelCased`.

Reply:
Fixed